tests/www/test_views.py

```diff
@@ -883,7 +882,7 @@ def test_run_with_not_runnable_states(self, get_default_executor_function):
             self.check_content_in_response('', resp, resp_code=200)
 
             msg = "Task is in the &#39;{}&#39; state which is not a valid state for execution. " \
-                  .format(state) + "The task must be cleared in order to be run"
+                .format(state) + "The task must be cleared in order to be run"
```

Comment:
You have a lot of whitespace-only/formatting-only changes here.

Please remove them from this PR (having unrelated changes in like this in a PR makes it harder to back-port fixes.)